src/connectors/snowflake/trulens/connectors/snowflake/connector.py

```diff
@@ -180,6 +181,22 @@ def _init_with_snowpark_session(
                 init_server_side_with_staged_packages,
             ).set_up_all()
 
+        # Add "trulens_workspace_version" tag to the current schema
```

Comment:
note that this will change the tag whenever you connect. Is that what you want? That is, consider these two cases:

1. If two ppl have two different trulens versions and are working together on the project the version will keep flip-flopping between the two whenever they connect.
2. I think if you fail the db revision check this will still execute even though it's not even at the right version.

Reply:
At high level, we don't care about the tag value, we care if tags exist to point to objects. If you think we can use another tag that is more constant than versions, we could use that too. We don;t have a workspace name like "app_name" to use that instead.

We have the following options:
1. version (with the current problems) you mentioned 
2. Ignore value - just use a constant value for the tag. 
3. Include metadata of tables under the schema like (number of apps) to help reduce querytime. Complex implementation.

In this PR i chose (1), but I think we could also go with (3).

Reply:
can we have the alembic version?

Reply:
Done